### ISSUE TITLE
Error querying Planet Staccato

### ISSUE THREAD

**Issue by philvarner** [kbgg/qgis-stac-browser#16]
Version: dev/f5a35fba113c72129efc397a8a036ca5b8080af8

Repro:
1. Open STAC dialog
2. check Planet[Staccato]
3. Search button
 
```
AttributeError: 'NoneType' object has no attribute 'extent' 
Traceback (most recent call last):
  File "/Users/pvarner/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/stac_browser/ui/query_dialog.py", line 59, in on_search_clicked
    self.get_time_period())
  File "/Users/pvarner/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/stac_browser/stac_browser.py", line 51, in on_search
    extent_rect = extent_layer.extent()
AttributeError: 'NoneType' object has no attribute 'extent'
```

The extent attribute makes me think that the Extent dialog shouldn't be empty?

<img width="719" alt="Screen Shot 2019-06-14 at 2 10 29 PM" src="https://user-images.githubusercontent.com/124996/59529156-3966c180-8eae-11e9-8d9b-0a22ac500ffb.png">

**Comment by kbgg** [comment]
Errors are still not being handled, this will be implemented with #15

**Comment by ymoisan** [comment]
I suspect "Error: Extent layer is not valid" that happens when EPSG != 4326 will either be handled with a [more descriptive error message](https://github.com/kbgg/qgis-stac-browser/blob/e94b70376160a77f8ddabf3514297302337d9ad8/stac_browser.py#L78) or that the API will allow other EPSG codes ?
